server/channels/db/migrations/postgres/000136_create_attribute_view.up.sql

```diff
@@ -0,0 +1,13 @@
+DROP MATERIALIZED VIEW IF EXISTS AttributeView;
+
+CREATE MATERIALIZED VIEW AttributeView AS
+  SELECT
+	pv.GroupID,
+	pv.TargetID,
+	pv.TargetType,
+	jsonb_object_agg(pf.Name, pv.Value) AS Attributes
+  FROM PropertyValues pv
+  LEFT JOIN PropertyFields pf ON pf.ID = pv.FieldID
+  GROUP BY pv.GroupID, pv.TargetID, pv.TargetType;
+
+REFRESH MATERIALIZED VIEW AttributeView;
```

Comment:
You don't need to refresh a mat view immediately after creating it. Creation of a mat view itself will populate it.